src/Graphics/AnimationFlashing.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _ANIMATION_FLASHING_H_
+#define _ANIMATION_FLASHING_H_
+
+#include "Animation.h"
+
+class AnimationFlashing : public Animation {
```

Comment:
look at this beautiful object-oriented structure

Reply:
polymorphin